src/examples/Colors/ColorsTemplate.js

```diff
@@ -7,437 +7,45 @@ const ColorBubble = ({colorClass}) => {
-                <ColorBubble colorClass='bg-yellow-100' />
-                <ColorBubble colorClass='bg-yellow-200' />
-                <ColorBubble colorClass='bg-yellow-300' />
-                <ColorBubble colorClass='bg-yellow-400' />
-                <ColorBubble colorClass='bg-yellow-500' />
-                <ColorBubble colorClass='bg-yellow-600' />
-                <ColorBubble colorClass='bg-yellow-700' />
-                <ColorBubble colorClass='bg-yellow-800' />
-                <ColorBubble colorClass='bg-yellow-900' />
-                <ColorBubble colorClass='bg-yellow-950' />
-                <ColorBubble colorClass='bg-yellow-1000' />
-            </div>
-
-            <div className='flex'>
-                <ColorBubble colorClass='bg-lime-50' />
-                <ColorBubble colorClass='bg-lime-100' />
-                <ColorBubble colorClass='bg-lime-200' />
-                <ColorBubble colorClass='bg-lime-300' />
-                <ColorBubble colorClass='bg-lime-400' />
-                <ColorBubble colorClass='bg-lime-500' />
-                <ColorBubble colorClass='bg-lime-600' />
-                <ColorBubble colorClass='bg-lime-700' />
-                <ColorBubble colorClass='bg-lime-800' />
-                <ColorBubble colorClass='bg-lime-900' />
-                <ColorBubble colorClass='bg-lime-950' />
-                <ColorBubble colorClass='bg-lime-1000' />
-            </div>
-
-            <div className='flex'>
-                <ColorBubble colorClass='bg-mint-50' />
-                <ColorBubble colorClass='bg-mint-100' />
-                <ColorBubble colorClass='bg-mint-200' />
-                <ColorBubble colorClass='bg-mint-300' />
-                <ColorBubble colorClass='bg-mint-400' />
-                <ColorBubble colorClass='bg-mint-500' />
-                <ColorBubble colorClass='bg-mint-600' />
-                <ColorBubble colorClass='bg-mint-700' />
-                <ColorBubble colorClass='bg-mint-800' />
-                <ColorBubble colorClass='bg-mint-900' />
-                <ColorBubble colorClass='bg-mint-950' />
-                <ColorBubble colorClass='bg-mint-1000' />
-            </div>
-
-            <div className='flex'>
-                <ColorBubble colorClass='bg-sky-50' />
-                <ColorBubble colorClass='bg-sky-100' />
-                <ColorBubble colorClass='bg-sky-200' />
-                <ColorBubble colorClass='bg-sky-300' />
-                <ColorBubble colorClass='bg-sky-400' />
-                <ColorBubble colorClass='bg-sky-500' />
-                <ColorBubble colorClass='bg-sky-600' />
-                <ColorBubble colorClass='bg-sky-700' />
-                <ColorBubble colorClass='bg-sky-800' />
-                <ColorBubble colorClass='bg-sky-900' />
-                <ColorBubble colorClass='bg-sky-950' />
-                <ColorBubble colorClass='bg-sky-1000' />
-            </div>
+        {colorClasses.map(colorClass => (
```

Comment:
also, i think you need to group them, 
so you might actually need a 2-dimensional array, check out the storybook entry and compare the before and after scenario and add a screenshot to the PR since this Is a visual update

Reply:
just pushed an updated, can you help confirm if this is what you meant